tests/correctness/exponent.futil

```diff
@@ -0,0 +1,24 @@
+import "primitives/std.lib";
+import "primitives/bitnum/math.futil";
+component main() -> () {
+  cells {
+    base = std_const(32,3);
+    exp = std_const(32,2);
+    b_x = std_mem_d1(32, 1, 1);
```

Comment:
I would call this "out" if this is the output memory. Also, would it make sense to make the inputs (base and exp) be read from other memories?

Reply:
`init` doesn't seem to use `1` as its done condition?